packages/nodes-base/nodes/Wait/Wait.node.ts

```diff
@@ -47,8 +202,27 @@ export class Wait extends Webhook {
 			{
 				...defaultWebhookDescription,
 				responseData: '={{$parameter["responseData"]}}',
-				path: '={{$parameter["options"]["webhookSuffix"] || ""}}',
+				path: webhookPath,
+				restartWebhook: true,
+			},
+			{
+				name: 'default',
+				httpMethod: '={{$parameter["resume"] === "form" ? "GET" : ""}}',
```

Comment:
can we default this to `GET` for waiting webhooks as well, so that we don't need [this](https://github.com/n8n-io/n8n/pull/7571/files#diff-881a47a009802648a246d1eb6b52527eb0307d3c8d12fa799b277b4ec2ccde60R937-R940) additional check.

Reply:
that check was mostly to avoid spam of errors to console, because when form operation selected webhook httpMethos would be undefined 
https://github.com/n8n-io/n8n/blob/c36ee23e5d68196467f865a1f568dd0e9dab5a0e/packages/nodes-base/nodes/Webhook/description.ts#L3-L16